cms/static/js/spec/views/course_video_settings_spec.js

```diff
@@ -92,6 +100,19 @@ define(
                 expect($preferanceContainerEl.find('.error-info').html()).toEqual(requiredText);
             };
 
+            selectPreference = function(preferenceSelector, preferanceValue) {
+                var $preference = $courseVideoSettingsEl.find(preferenceSelector);
+                // Select a vlaue for preference.
+                $preference.val(preferanceValue);
+                // Trigger on change event.
+                $preference.change();
+            };
+
+            chooseProvider = function(selectedProvider) {
```

Comment:
Can we rename this? This is selecting a provider + verifying the selected value.